Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+# Cardano Certification Standard
+
+**Draft version**
+**Last modification September 21th 2023**
+**Authors:**
+
+- Romain Soulat
+
+## 1 Introduction
+
+The Cardano Certification Standard (CCS) is a framework designed to facilitate the comparability of independent evaluations within the Cardano blockchain ecosystem. This standard establishes a comprehensive set of requirements for assessing the security features of Decentralized Applications (DApps) on the Cardano blockchain, as well as the assurance measures applied during security evaluations.
+
+The primary objective of the evaluation process is to establish a level of confidence that the security functionalities of these DApps and the assurance measures align with the predefined requirements. Consequently, the evaluation results serve as a valuable resource for end-users seeking to gauge whether a particular DApp aligns with their security requirements and expectations.
```

Comment:
"align with the requirements defined here" otherwise it might be confused with particular requirements for particular DApps